wiki/BanchoBot/ja.md

```diff
@@ -8,23 +8,23 @@ tags:
 
 # BanchoBot
 
-*その他の用途については、[Bancho（曖昧さ回避）](/wiki/Disambiguation/Bancho) を参照してください。*
+その他の用途については、[Bancho（曖昧さ回避）](/wiki/Disambiguation/Bancho) を参照してください。
 
 ![BanchoBot ユーザーカード](img/BanchoBot.jpg "BanchoBot ユーザーカード")
 
-**BanchoBot**（*Bancho*と呼ばれることもあります）はosu!のために作られたオンラインチャットボットで、ゲームに関するメッセージ（例：累計プレイ回数、リトライ回数など）をお知らせしたり、特定のコマンドに応答したりして、ゲーム内のチャットでプレイヤーを支援します。[Echo](https://osu.ppy.sh/users/431)によってプログラムされ、[Bancho IRC](/wiki/Community/Internet_Relay_Chat) (Internet Relay Chat)を提供しています。
+**BanchoBot**（Banchoと呼ばれることもあります）はosu!のために作られたオンラインチャットボットで、ゲームに関するメッセージ（例：累計プレイ回数、リトライ回数など）をお知らせしたり、特定のコマンドに応答したりして、ゲーム内のチャットでプレイヤーを支援します。[Echo](https://osu.ppy.sh/users/431)によってプログラムされ、[Bancho IRC](/wiki/Community/Internet_Relay_Chat) (Internet Relay Chat)を提供しています。
```

Comment:
No need to remove `* *` from Bancho, since the original article has it and It works normally in Japanese one also.